project-reports/2023/2023-Q2-Hyperledger-Indy.md

```diff
@@ -0,0 +1,155 @@
+[Hyperledger AnonCreds]: https://github.com/hyperledger/anoncreds-rs
+
+### **Specifications**
+
+- [did:indy Specification], [did:indy Specification source repository]
+- [did:indy Networks]
+- [Indy HIPE] (Hyperledger Indy Project Enhancements)
+
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+
+## Project Health
+
+The project is relatively healthy, with a moderate number of updates
+from a range of organizations. An exercise in the Indy Contributors
+call resulted in an [Indy Roadmap] defining the direction of the project
+the community would like to see.
+
+[Indy Roadmap]: https://hackmd.io/GeRP00i0Sj-7z4zXn2MB5g?view
+
+Per the [Indy Quarterly Activity Dashboard], there were 97 commits
+(the same as last quarter) from 24 contributors (almost double last quarter).
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Findy/dashboard;subTab=technical?time=%7B%22from%22:%222023-01-01T08:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-03-31T07:00:00.000Z%22%7D
+
+## Questions/Issues for the TSC
+
+None
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+- indy-node (Ubuntu 20.04) – v1.13.2-rc5, v1.13.2-rc6
+- indy-plenum (Ubuntu 20.04) - v1.13.1-rc4
+- indy-vdr - v0.3.4, v0.4.0-dev.1 to v0.4.0-dev.12
+- indy-node-container - v1.2.4
+- indy-test-automation - v0.9.0, v0.9.1, v0.9, v0.9.2, v0.10.0, v0.11.0, v0.11, v0.11.1
+- indy-cli-rs - v0.1.0
+- indy-shared-rs - v0.3.2
+- indy-sdk-react-native - v0.3.1
+
+## Overall Activity in the Past Quarter
+
+Much of the progress this quarter focused on [indy-vdr] and the movement of
+Aries Frameworks from using the [indy-sdk] to using the so-called shared
+components, including [indy-vdr], [aries-askar] and [anoncreds-rs]. The move
+away from the [Indy SDK] triggered updates and releases of a lot of Indy
+artifacts from a variety of repositories. See the list of releases
+[here](#releases). A significant conversion of a deployed agent (an Aries
+mediator) from the Indy SDK to Askar once again demonstrated the performance and
+stablity benefits of such a change.
+
+Further progress was made on the Indy CI/CD pipeline automation and on the new
+Ubuntu 20.04 release. Work on completing that task has been slowed because of
+the nature of the final task -- deciding what changes in the Indy Node/Indy
+Plenum `ubuntu-16.04` (formerly `main`) and `stable` branches need to be
+cherry-picked onto the new new (Ubuntu 20.04) `main` branch. The nature of the
+work restricts those capable of making the decisions, and those folks are busy
```

Comment:
Q: For the understanding, is there a set deadline or other form of important timelines, by when the Indy project would need to complete migration to Ubuntu 20.04?

Reply:
16.04 has been EOL, so we need this as quickly as we can get the resources to complete the work.

Reply:
Just trying to understand the Ubuntu challenges and plans because this is often a question across projects...

It looks like many of the Indy release GitHub Actions use ubuntu-latest, what is still stuck on 16.04?
At least in Fabric, we are using Ubuntu 20.04 (rather than ubuntu-latest) so that the built artifacts work on both 20.04 and 22.04 (we've found the reverse is not always true).
Do you have different branches for different Ubuntu releases?